### ISSUE TITLE
Force registration/login before you can share 

### ISSUE THREAD

**Issue by corinnewo** [ji-devs/ji-cloud#3273]
If you are not logged in yet and click the **Share** icon either on card on content page, or within the JIG, force the user to login first.
@ChaykeeMor - need text that is friendly - to tell the user that it is time to login if they want to share content.
https://zpl.io/7w6MGyj 

Please add the text here, and then we can give to @MendyBerger to implement.

**Comment by ChaykeeMor** [comment]
@corinnewo is this taken care of with #3274?

**Comment by corinnewo** [comment]
No, this is separate - same popup, but this is for sharing. The other was after 5 JIG views.

**Comment by corinnewo** [comment]
@ChaykeeMor No, this is separate - same popup, but this is for sharing. The other was after 5 JIG views.

I have updated the link to the most recent Zeplin: https://zpl.io/llRGJKk, but @GaliShapira must still update the text in Zeplin according to your comment.

**Comment by ChaykeeMor** [comment]
k @corinnewo We need one for clicking on create. Right now it just bumbs you to full page login and it could use the generic pop up as well....time to log in!
